src/lib/Microsoft.Health.Common/Extension/ExceptionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Common.Extension
+{
+    public static class ExceptionExtensions
+    {
+        private const string LogToCustomerAttribute = "LogToCustomer";
+
+        /// <summary>
+        /// Flags the exception as being logged to the customer or not.
+        /// </summary>
+        /// <param name="error">The root exception</param>
+        /// <param name="logToCustomer">Boolean to set whether the exception should be logged to third-party customers</param>
+        public static void SetShouldLogToCustomer(this Exception error, bool logToCustomer)
```

Comment:
Suggestion: You can update this to return Exception and allow you to use the extension in a fluent syntax.  I.E.
```
new Exception("Error")
.SetShouldLogToCustomer()
.OtherMethodHere();
```